### DISCUSSION TITLE
Grading scheme assignments/lab

### DISCUSSION THREAD

**Discussion by LachezaraLaz** [SOEN345-WINTER2024/discussion#9] [General]
In the course outline, the grading scheme says 20% for assignments including labs, does that mean all assignments and labs have the same weight or assignments are 10% and the labs are the other 10%? In other words, how is the grade distributed in that 20% between assignments and labs?

**Comment by shtest1** [comment]
each assignment will have 5% with 2 assignment, the remaining will be the lab assignments.
